core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,77 @@
 <script type="text/ng-template" id="components/socialButtons">
-  <div class="oppia-footer-social">
+  <div class="oppia footer social">
     <div>
       <h4>
         <span translate="I18N_FOOTER_FOLLOW_US"></span>:
       </h4>
     </div>
-    <a href="https://plus.google.com/109898456505810251700/about" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/gplus.png')]>" alt="Google+">
-    </a>
-    <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/youtube.png')]>" alt="YouTube">
-    </a>
-    <a href="https://www.facebook.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/fb.png')]>" alt="Facebook">
-    </a>
-    <a href="https://twitter.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/twitter.png')]>" alt="Twitter">
-    </a>
+    
+    <div class="oppia footer icons">
+
+        <div class="Google Plus">
```

Comment:
The first class hold the Google, Facebook, Youtube and twitter icons in place, as block.
The sub divs are individual sqaured boxes.